### PR TITLE
Index upgrade known issue

### PR DESCRIPTION
**Pull request by ggray-cb**
PR for the Index Server upgrade issue known issue. 

[Here's the preview.](https://preview.docs-test.couchbase.com/index-upgrade-known-issue/server/current/release-notes/relnotes.html#known-issues-761)

### CODE REVIEW

**Review by bfavini — APPROVED** [review]
Approved. (Verbal approval from @dave-finlay and @SaptarshiSen-CB )

**Comment by dave-finlay** [comment]
one second - I have a comment - if you could hold off merging

File: modules/release-notes/partials/index-upgrade-issue.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+==== Index Service 
+[#table-known-issues-760-index-service, cols="10,40,40"]
+|===
+|Issue | Description | Workaround
+
+| https://issues.couchbase.com/browse/MB-62547[MB-62547]
+a| A critical issue occurs when using the offline-upgrade, graceful failover, or delta-recovery upgrade methods on Index Service nodes from Couchbase Server versions 7.1 through 7.2 to versions 7.6.0 or 7.6.1.
```
Comment (dave-finlay) [inline modules/release-notes/partials/index-upgrade-issue.adoc]: Could you change as follows:

> A critical issue occurs when using the offline-upgrade or graceful failover/ delta-recovery upgrade methods on Index Service nodes from Couchbase Server versions 7.1 through 7.2 to versions 7.6.0 or 7.6.1.

**Review by dave-finlay — COMMENTED** [review]
One minor change to the text of the warning. Else looks great. Thanks Gary and Beth.

File: modules/release-notes/partials/index-upgrade-issue.adoc (hunk 1 above)
Comment (dave-finlay) [reply modules/release-notes/partials/index-upgrade-issue.adoc]: ("Graceful failover / delta-recovery" is an upgrade method all by itself.)

**Pull request merged**